verl/trainer/config/_generated_ppo_trainer.yaml

```diff
@@ -286,6 +286,8 @@ actor_rollout_ref:
     use_fused_kernels: false
     fused_kernel_options:
       impl_backend: torch
+    attn_implementation: flash_attention_2
```

Comment:
currently, for verl default hf `attn_implementation` etc. accessible by `override_config: {}` few lines above